Providers/Profile.swift

```diff
@@ -296,7 +296,7 @@ public class BrowserProfile: Profile {
         let commands = items.map { item in
             SyncCommand.fromShareItem(item, withAction: "displayURI")
         }
-        self.remoteClientsAndTabs.insertCommands(commands, forClients: clients)
+        self.remoteClientsAndTabs.insertCommands(commands, forClients: clients) >>> { self.syncManager.syncClients() }
```

Comment:
Do we need to explicitly denote a background operation here (e.g., if we're called from an extension)?

Please also test this with a queued tab ready to receive. I believe the stub delegate (that we'll use if this is called from inside an extension) just throws away the incoming `displayURI` command, and the received command will never be received.

This is the first time we'll be triggering a sync from an extension, so there are some new things to address…